frontend/src/reducers/users.js

```diff
@@ -23,6 +23,8 @@ export default function users(state={
 
   switch (type) {
     case constants.USER_GROUPS_SUCCESS:
+    Object.keys(groups).map(value =>
+      groups[value] = Object.assign({}, groups[value], {balance: groups[value].balance/= 100}))
```

Comment:
Is `Object.assign` polyfilled by babel? Not supported in IE11
http://kangax.github.io/compat-table/es6/#test-Object_static_methods_Object.assign